project.make

```diff
@@ -639,7 +639,7 @@ libraries[smart-app-banner][destination] = "libraries"
 
 ; For ding_react.
 libraries[ddb-react][download][type] = "get"
-libraries[ddb-react][download][url] = https://github.com/danskernesdigitalebibliotek/ddb-react/releases/download/latest/dist.zip
+libraries[ddb-react][download][url] = https://github.com/reload/ddb-react/releases/download/2.3.1-beta1/dist.zip
```

Comment:
Should use the code base at https://github.com/danskernesdigitalebibliotek/ddb-react

Reply:
Agreed but I need this change for the PR environment to reflect the latest changes. I have updated the text accordingly.